main.tf

```diff
@@ -45,7 +45,7 @@ resource "azurerm_application_insights" "appinsights" {
   count = "${var.appinsights_instrumentation_key == "" ? 1 : 0}"
 
   name                = "${var.product}-appinsights-${var.env}"
-  location            = "${var.appinsights_location}"
+  location            = "${var.location}"
```

Comment:
this should probably be an option and keep to west europe by default

Reply:
Technically it can already do that, with some doc updates.

How about two stages:
1. Unset the default, so it has to be explicitly configured. Lots of PRs later.
2. Default to 'UK South' so new deployments do the right thing.

I guess it's the churn of every app needing to PR that I don't like.  Those on shared appinsight would be fine.

Reply:
What happens if you change the default and one is already created?

Reply:
Changing location will create a new instance according to the docs.  So loss of historic appinsight data.

Reply:
I would say no point changing this now, 
we do need a module written at some point so disable the emails going to all owners and contributors. currently only possible through arm afaik

Reply:
I think I agree.  I'll change this PR into purely a doc cleanup.  But leave the default as is.

Reply:
sound good just change everything to point to UK South except the default value